src/remix.ts

```diff
@@ -0,0 +1,34 @@
+import type { AppLoadContext } from '@remix-run/cloudflare'
+import type { Context } from 'hono'
+
+export type GetLoadContext = (args: {
+  request: Request
+  context: {
+    // Relaxing the type definition
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    cloudflare: any
+  }
+}) => AppLoadContext
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const defaultGetLoadContext = ({ context }: any) => {
```

Comment:
Names are hard, but if you gave that type a name like:

```ts
type SomeContext = {
  cloudflare: {
    env: Record<string, unknown>
    cf: IncomingRequestCfProperties
    ctx: ExecutionContext
    caches: typeof caches
  }
};
```

We could use it like

```ts
export type GetLoadContext = (args: {
  request: Request
  context: SomeContext
}) => AppLoadContext

export const defaultGetLoadContext = ({ context }: { context: SomeContext}) => {
```

Reply:
If you override the definition of `AppLoadContext` in your app: https://github.com/yusukebe/hono-remix-adapter/pull/6/files#diff-fe3b5413a3ab56944fc97a3eb9b71e83c41e4d0b41a2303932ae92bd8c82cc8fR11-R13

The type mismatch will occur. To remove the typo error, I've added the `any`. This may be fixed in the future, but it remains as `any`.

Reply:
@yusukebe Oh, right. Apologies for not testing that locally. I think 0.2.0 shipped with this type, because I'm getting this type error in my app after updating

Nothing a `@ts-expect-error` can't fix, though

<img width="1780" alt="Screenshot 2024-09-23 at 7 46 38 AM" src="https://github.com/user-attachments/assets/f4086423-fb93-4504-9580-589ee394ed8c">

<img width="726" alt="Screenshot 2024-09-23 at 7 40 38 AM" src="https://github.com/user-attachments/assets/35408a94-0de9-4705-9638-971f1fe96480">

Reply:
@jfsiii 

Oooops. Sorry. The building way was wrong. Fixed it and released the new version. Try `v0.2.1`.

Reply:
No worries. It happens. And it was kind of my fault, anyway.

Thanks!